api.planx.uk/server.ts

```diff
@@ -285,6 +286,10 @@ assert(process.env.UNIFORM_TOKEN_URL);
 assert(process.env.UNIFORM_SUBMISSION_URL);
 app.post("/uniform/:localAuthority", useHasuraAuth, sendToUniform);
 
+app.post("/email-submission/:localAuthority", useHasuraAuth, sendToEmail);
+
+app.get("/download-application-files/:sessionId", downloadApplicationFiles);
```

Comment:
Something to note: This PR cements the fact that a Send component is what defines Save & Return. There's no way we could have a service that sends to email but does not use the Save & Return frontend - even if it was a simple or internal service created with PlanX. 

Previously we could have used the Pay component or allowed Editors to turn this on or off for a flow, but that changes going forward from here.

I actually don't think this is actually our intention - I'll add this to the dev call agenda as it's an interesting one.

I think the intention is to give users Save & Return frontend for complex flows that are likely to require multiple visits to the site.

Using S3 pre-signed urls (see other comments) could help us keep this door open.

Not sure how clear this is....! 😅 Let's try to talk about it though.

Reply:
Thanks for raising this question! Was productive to chat about in the dev call yesterday, here's my attemt at summarizing the main points that came up there:

- For this send to email feature, even when used within very short services in the future, we will still always need to know the applicant's email address that the council would reply to - so I actually think that still leading with the email prompt makes a lot of sense! It also has consistent passport storage, which would save us future mappings of needing to track something like `applicant.email` || `siteContact.email` || `applicant.agent.email` and so on _per service_ 
- For now, the "Send" component still feels like a good proxy for something that is an "application", eg data is collected, sent away, and processed. Whereas services without "Send" can be thought of as more computational, eg answer questions and receive a personalised informational result. 
  - "Save & Return" is a useful disctintion for defining application data retention - applications are stored in `lowcal_sessions` (within 7-28 day time limits) versus local storage for non-applications
  - Even if a short discretionary service using send to email will always be completed in a single sitting, having Save & Return enabled shouldn't be a hinderance currently - it simply means that applicant will be less aware of reminder/expiry emails etc

Reply:
Thanks for that summary - good to have it captured here 👍 Agreed with all of the above!